RELEASE.txt

```diff
@@ -18,64 +18,82 @@ make a PR to update these notes after you are done with the release. ;-)
 
   1. Review and cleanup ``doc/release/release_dev.txt``.
 
-  2. Make a list of merges and contributors by running
+  2. Make a list of merges, contributors, and reviewers by running
      ``doc/release/contribs.py <tag of previous release>``.
 
-  3. Paste this list at the end of the ``release_dev.txt``. Scan the PR titles
-     for highlights, deprecations, and API changes, and mention these in the
-     relevant sections of the notes.
+  3. Paste this list at the end of the ``release_dev.txt``.
 
-  4. Rename to ``doc/release/release_<major>.<minor>.txt``
+  4. Scan the PR titles for highlights, deprecations, API changes,
+     and bugfixes, and mention these in the relevant sections of the notes.
+     Try to present the information in an expressive way by mentioning
+     the affected functions, elaborating on the changes and their
+     consequences. If possible, organize semantically close PRs in groups.
 
-  5. Copy ``doc/release/release_template.txt`` to
+  5. Rename the file to ``doc/release/release_<major>.<minor>.txt``
+
+  6. Copy ``doc/release/release_template.txt`` to
      ``doc/release/release_dev.txt`` for the next release.
 
-  6. Copy relevant deprecations from ``release_<major>_<minor>.txt``
+  7. Copy relevant deprecations from ``release_<major>_<minor>.txt``
      to ``release_dev.txt``.
 
-- Update the version number in ``skimage/__init__.py`` and commit.
+- Submit the release notes for review by other project maintainers:
+
+  - Create a temporary PR from v<major>.<minor>.x branch to master
+    (at this point, the difference should show the full contents of the
+     release notes).
+  - Discuss, and make the changes directly to v<major>.<minor>.x branch.
+  - Once the consensus is found, close the PR and go to the next step.
```

Comment:
With *close the PR* you mean the PR should be merged to master, or not?  When does this file find its way back into master, I'm not sure we've added that to the instructions.

Reply:
Good point! I think the most convenient option is to actually merge the PR. The latest commit reorders the steps a bit so that the merge is non-destructive.